### PR TITLE
Sound

### PR DESCRIPTION
**Pull request by linxcat**
@JWingWangWong could you please replace all the tabs you added? They are supposed to be spaces and it looks awful.

### CODE REVIEW

**Comment by JWingWangWong** [comment]
Fixed tabs to spaces.

File: Assets/Scripts/Interaction/Hand.cs [hunk 1]
```diff
@@ -23,6 +23,31 @@ public class Hand : MonoBehaviour {
     Sword sword;
     Transform teleLineSpawn;
     MagicDraw magicDraw;
+    Spells spells;using System;
```
Comment (linxcat) [inline Assets/Scripts/Interaction/Hand.cs]: This is a copy/paste problem, from what I can tell.

File: Assets/Scripts/Interaction/Hand.cs [hunk 2]
```diff
@@ -155,8 +180,160 @@ public void setLock(bool value)
     }
 
     public void chargeSword(bool charge) {
-        if (charge) sword.StartCoroutine("swordCharge");
-        else sword.StopCoroutine("swordCharge");
+        if (charge) {
```
Comment (linxcat) [inline Assets/Scripts/Interaction/Hand.cs:158]: There are a lot of changes that shouldn't be changes, could you check to make sure there isn't a problem here?

File: Assets/Scripts/Interaction/MagicDraw.cs [hunk 3]
```diff
@@ -6,12 +6,19 @@ public class MagicDraw : MonoBehaviour {
...
 
     bool isDrawing = false;
 
-	void Awake () {
+    void Start() {
+        audioSource = GetComponent<AudioSource>();
+    }
+
+
+    void Awake () {
```
Comment (linxcat) [inline Assets/Scripts/Interaction/MagicDraw.cs]: Awake should come before Start, by convention.

File: Assets/Scripts/Interaction/InputHandler.cs [hunk 4]
```diff
@@ -100,10 +100,14 @@ void switchHandMode()
         if (truePrimary) leftHand.switchDebug();
         else rightHand.switchDebug();
     }
-    
+
     void swordCharge(bool charge) {
-        if (truePrimary) leftHand.chargeSword(charge);
-        else rightHand.chargeSword(charge);
+      if (truePrimary) {
```
Comment (linxcat) [inline Assets/Scripts/Interaction/InputHandler.cs]: could you return this to the two-line version, please?

**Review by JWingWangWong — DISMISSED** [review]

**Review by linxcat — COMMENTED** [review]
code review

File: Assets/Scripts/Interaction/Hand.cs (hunk 1 above)
Comment (JWingWangWong) [reply Assets/Scripts/Interaction/Hand.cs]: Fixed.

File: Assets/Scripts/Interaction/InputHandler.cs (hunk 4 above)
Comment (JWingWangWong) [reply Assets/Scripts/Interaction/InputHandler.cs]: reverted

File: Assets/Scripts/Interaction/Hand.cs (hunk 2 above)
Comment (JWingWangWong) [reply Assets/Scripts/Interaction/Hand.cs:158]: I had to add a stop sound here to ensure that the charge sound wouldn't go if the user stopped charging

File: Assets/Scripts/Interaction/MagicDraw.cs (hunk 3 above)
Comment (JWingWangWong) [reply Assets/Scripts/Interaction/MagicDraw.cs]: Now before start.

**Review by JWingWangWong — COMMENTED** [review]
Fixed/answered all the code review requests.

**Comment by linxcat** [comment]
good stuff

**Pull request merged**